doc-Installing_on_OpenShift_Container_Platform/topics/configuring_external_auth_ocp.adoc

```diff
@@ -0,0 +1,292 @@
+It is based on the auth `httpd` container and generates the `httpd` `authconfig` map
+needed to enable external authentication.
+
+
+===== Preparing to Deploy the `httpd-configmap-generator` Application
+
+The `cfme-httpd-configmap-generator` image is obtained from the Red Hat Container Catalog at
+https://access.redhat.com/containers/?tab=images&platform=openshift#/registry.access.redhat.com/cloudforms46/cfme-httpd-configmap-generator.
+
+To obtain the latest container image, run:
+
+----
+oc import-image my-cloudforms46/cfme-httpd-configmap-generator --from=registry.access.redhat.com/cloudforms46/cfme-httpd-configmap-generator --confirm
+----
+
+The `httpd-configmap-generator` service account must be added to the `httpd-scc-sysadmin` SCC before the `httpd-configmap-generator` can run.
+
+. Log in to OpenShift as an admin user: 
++
+----
+$ oc login -u <user> -p <password>
+----
++
+. As the admin user, create the `httpd-scc-sysadmin` SCC:
++
+----
+$ oc create -f templates/httpd-scc-sysadmin.yaml
+----
++
+. Add the `httpd-configmap-generator` service account to the `httpd-scc-sysadmin` SCC:
++
+----
+$ oc adm policy add-scc-to-user httpd-scc-sysadmin system:serviceaccount:<your-namespace>:httpd-configmap-generator
+----
++
+. Verify that the `httpd-configmap-generator` service account is now included in the `httpd-scc-sysadmin` SCC:
++
+----
+$ oc describe scc httpd-scc-sysadmin | grep Users
+Users:        system:serviceaccount:<your-namespace>:httpd-configmap-generator
+----
+
+
+===== Deploying the `httpd-configmap-generator` Application
+
+. As a regular user, run:
++
+----
+$ oc create -f httpd-configmap-generator-template.yaml
+----
++
+. Verify the template is available with your project:
++
+----
+$ oc get templates
+NAME                        DESCRIPTION                                 PARAMETERS     OBJECTS
+httpd-configmap-generator   Httpd Configmap Generator                   6 (all set)    3
+----
++
+. Deploy the `httpd-configmap-generator`:
++
+----
+$ oc new-app --template=httpd-configmap-generator
+----
++
+. Check the readiness of the `httpd-configmap-generator`:
++
+----
+$ oc get pods
+NAME                                READY     STATUS    RESTARTS   AGE
+httpd-configmap-generator-1-txc34   1/1       Running   0          1h
+----
+
+===== Getting the Pod Name
+
+To work with the `httpd-configmap-generator` script in the `httpd-configmap-generator` pod, it is necessary to
+get the pod name as below:
+
+----
+$ CONFIGMAP_GENERATOR_POD=`oc get pods | grep "httpd-configmap-generator" | cut -f1 -d" "`
+----
+
+
+===== Generating an `authconfig` Map for External Authentication Against IPA
+
+The following example shows how to generate a configmap for external authentication against IPA.
+
+. To generate an `authconfig` map for external authentication against IPA, run
++
+----
+$ oc rsh $CONFIGMAP_GENERATOR_POD httpd_configmap_generator ipa \
```

Comment:
this command will probably fail in CloudForms if it isn't fixed yet. it requires 'bash -c'. 
https://github.com/ManageIQ/httpd_configmap_generator